content/jobs/seniorjs2021.md

```diff
@@ -1,7 +1,7 @@
 ---
-position: open
+position: filled
 banner: Senior JavaScript Developer
-sub: Collaborating on our front-end development process
+sub: This position has been filled
```

Comment:
I see that this is the language we've used on other postings - but since we're closing the applications before we've actually filled the position, it could be surprising to people still in the process? Maybe we should say something like "No longer accepting applications for this position"?

Reply:
sounds good to me , i'll apply these changes

Reply:
resolved by https://github.com/oddbird/oddleventy/pull/185/commits/74405df3b2b881b5de05c2c7253e59852a23e8fa